plugins/modules/rds_global_cluster_info.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023 Ansible Project
+# Copyright (c) 2023 Gomathi Selvi Srinivasan (@GomathiselviS)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+module: rds_global_cluster_info
+version_added: 6.6.0
+short_description: Obtain information about Aurora global database clusters
+description:
+  - Obtain information about Aurora global database clusters.
+options:
+    global_cluster_identifier:
```

Comment:
This can be turned to a list

Reply:
Why do you think so? The API accepts only a string

Reply:
yes it accepts only a string, having a list is to avoid multiple calls on user side with different values of the identifier increasing the response time as for each one ansible need to build the env and the module needs to create a new session. But we can keep it like this for now